tutorials/polkadot-sdk/parachains/deploy-to-relay-chain/deploy-on-paseo.md

```diff
@@ -14,18 +14,22 @@ This tutorial demonstrates deploying a parachain on a public test network like t
 Before you start, you need to have the following prerequisites:
 
 - You know how to generate and modify chain specification files as described in the [Generate Chain Specs](/develop/parachains/deployment/generate-chain-specs){target=\_blank} section
+- You have installed the [`polkadot-omni-node`](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/omni_node/index.html){target=\_blank} and [`chain-spec-builder`](https://docs.rs/staging-chain-spec-builder/latest/staging_chain_spec_builder/){target=\_blank} binaries
```

Comment:
nit:

Referencing the Omni Node `reference_docs` doesn't help much with installation, for which users might search info inside them, given we suggest installing `polkadot-omni-node`. We can either stick to the reference docs (which don't help with installation, and people will have to figure out by themselves), or link speculatively to `https://crates.io/crates/polkadot-omni-node`, but then these docs might need to wait for the crates to be published.

[polkadot-omni-node crate](https://crates.io/crates/polkadot-omni-node) will show up [this readme](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/polkadot-omni-node/README.md) once the new major release is out and associated crates are released too. There are two options of installation: 
1. `cargo install polkadot-omni-node`
2.  `wget`'ting the binary artifact from the release: e.g. ` wget https://github.com/paritytech/polkadot-sdk/releases/download/polkadot-stable2412-rc2/polkadot-omni-node`.

Reply:
Better to also link the `crates.io` staging-chain-spec-builder, because once published will  present this info:  https://github.com/paritytech/polkadot-sdk/tree/master/substrate/bin/utils/chain-spec-builder#chain-spec-builder

```suggestion
- You have installed the [`polkadot-omni-node`](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/omni_node/index.html){target=\_blank} and [`chain-spec-builder`](https://crates.io/crates/staging-chain-spec-builder){target=\_blank} binaries
```

Reply:
I am fine with this, maybe one thing to do at the same time is open an issue to track that we need to update from READMEs to crates.io links once the crates are published.

Reply:
This sounds good. @kapetan3sid could you please take care of this in line with how you're currently doing issue tracking? appreciate it 🙏

Reply:
@iulianbarbu Do you think it's worth to add a quick line about installing from a specific release via git? (this is useful for macOS / Windows machines), i.e.,: 

```
cargo install --git https://github.com/paritytech/polkadot-sdk --tag <tag> --force polkadot-omni-node
```

Reply:
We'll have aarch64-darwin binaries shipped for omni-node (not for chain-spec-builder though) with the major release too. Not super sure if the polkadot-omni-node builds on Windows, I haven't tried it, but I haven't seen requests for it either.

All in all '--git' + '--tag' installation sounds good to me.

Reply:
ok awesome, thanks so much!